sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.1/async_samples/sample_authentication_v3_1_async.py

```diff
@@ -7,7 +7,7 @@
 # --------------------------------------------------------------------------
 
 """
-FILE: sample_authentication_async.py
+FILE: sample_authentication_v3_1_async.py
```

Comment:
Might want to double-check that these renames won't break any links in the MS docs